libr/core/cmd_anal.c

```diff
@@ -2621,6 +2625,8 @@ static void cmd_anal_esil(RCore *core, const char *input) {
 				}
 			} else eprintf ("esil.stats is empty. Run 'aei'\n");
 			break;
+		case '-':
+			sdb_foreach (esil->stats, sdb_del, esil);
```

Comment:
Agree. So just drop the db with an sdb.reset or so. That will be faster than a sdb.foreach+unset

> On 06 Aug 2016, at 19:00, Ahmed Mohamed Abd El-MAwgood <notifications@github.com> wrote:
> 
> In libr/core/cmd_anal.c:
> 
> > @@ -2621,6 +2625,8 @@ static void cmd_anal_esil(RCore *core, const char *input) {
> >  				}
> >  			} else eprintf ("esil.stats is empty. Run 'aei'\n");
> >  			break;
> > +		case '-':
> > +			sdb_foreach (esil->stats, sdb_del, esil);
> actually removing some keys wont make really any sense imho. the aek gives descriptive info about the emulation of the last emulated instruction. so you either delete it or keep it, right ?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>